src/Tizen.Applications.EventManager/Tizen.Applications/ApplicationEventManager.cs

```diff
@@ -0,0 +1,105 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Applications.EventManager
+{
+    /// <summary>
+    /// The EventManager Class provides functions to broadcast user-defined event.
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// public class EventManagerSample
+    /// {
+    ///     /// ...
+    ///     Bundle bundle = new Bundle();
+    ///     bundle.AddItem("key", "value");
+    ///     EventManager.PublishAppEvent("event.org.example.helloworld.event, bundle);
+    /// }
+    /// </code>
+    /// </example>
+    /// <since_tizen> 6 </since_tizen>
+    public static class ApplicationEventManager
+    {
+        /// <summary>
+        /// Sends the User-Event to receiver applications.
+        /// </summary>
+        /// <param name="eventName">The event's name to send.</param>
+        /// <param name="eventData"> The event's data to send.</param>
+        /// <remarks>The format of User-Event's name MUST be "event.{sender's appid}.{user-defined name}", refer to 'The name-format of User-Event' section,
+        /// If the event_name is invalid, throw InvalidOperationException</remarks>
+        /// <exception cref="ArgumentException">Thrown in case of an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown in case of any internal error.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void Publish(string eventName, Bundle eventData)
+        {
+            Interop.AppEvent.ErrorCode err = Interop.AppEvent.EventPublishAppEvent(eventName, eventData.SafeBundleHandle);
+            if (err != Interop.AppEvent.ErrorCode.None)
+            {
+                ErrorFactory.ThrowException(err, "Publish event");
+            }
+        }
+
+        /// <summary>
+        /// Sends the User-Event to trusted receiver-applications.
+        /// </summary>
+        /// <param name="eventName">The event's name to send.</param>
+        /// <param name="eventData"> The event's data to send.</param>
+        /// <param name="isTrusted"> Whether the event is trusted or not.</param>
+        /// <remarks>The application which has same certification with sender can receive the event.
+        /// The format of User-Event's name MUST be "event.{sender's appid}.{user-defined name}", refer to 'The name-format of User-Event' section,
```

Comment:
I don't understand why need to use this format. Why we add appid in event name?
If Application developer cheat a appid, it can send?

Reply:
esd(event system daemon) checks if the appid is valid.

Reply:
> esd(event system daemon) checks if the appid is valid.

"event.{sender's appid}." always a same strings. 
And We can predict the value "event.{sender's appid}." 
So, I thinks application developer did not need to include "event.{sender's appid}." prefix in EventName

App developer just pass a {user-define-name} and in platform make formated( "event.{sender's appid}.{user-defined name}") event name

Reply:
This spec is the same as native api. Then if we change this spec, app developers might be confused about using this api